content/en/Platform Deep Dive/Collaboration/Organization/Organization Settings/SAML SSO/saml-migration.md

```diff
@@ -0,0 +1,141 @@
+
+## How to Update Your SAML Configuration
+
+There are two ways to migrate to the new SAML implementation:
+
+- **Option 1**: Create a new SAML app with your identity provider (IdP). For details, see our [instructions](/platform-deep-dive/collaboration/organization/organization-settings/saml-sso/#general-configuration-workflow).
+- **Option 2**: Update your existing SAML configuration between Cobalt and your IdP, as described below.
+
+Before updating your existing configuration, note the following:
+
+- To avoid disrupting anyone's workflows, choose a time when the impact on users is minimal.
+- You need to copy parameters between two systems, so it's handy to have both applications open.
+- You can refer to [instructions for some popular IdPs](#instructions-for-specific-identity-providers) below.
+
+Here's a general workflow to **update your existing SAML configuration**:
+
+1. In your identity provider:
+    - Update the <span style="background-color: #ECE6FA; padding: 2px;">**ACS URL**</span> with the current value from Cobalt. The name of this parameter may differ for different IdPs. To get the ACS URL, in Cobalt, go to **Settings** > **Identity & Access**. Under **Configure SAML**, select **Configure**, and copy the ACS URL value.
+1. In Cobalt:
+    - Go to **Settings** > **Identity & Access**.
+      - Make sure that the **Enforce SAML** toggle is turned off. You can [enforce SAML](/platform-deep-dive/collaboration/organization/organization-settings/saml-sso/#enforce-saml-sso) after you complete the configuration.
+      - You can view your legacy configuration under **Legacy SAML Configuration**. It may not match your current setup in the IdP.
+    - Under **Configure SAML**, select **Configure**.
+    - Update the <span style="background-color: #ECE6FA; padding: 2px;">**IdP Certificate**</span> with the current value from your identity provider, and select **Save Configuration** to confirm.
+1. We no longer require a <span style="background-color: #ECE6FA; padding: 2px;">**RelayState**</span> within the assertion. Delete it from the configuration in your IdP.
+1. Test the configuration. In the identity provider, select the Cobalt tile. You should be authenticated to Cobalt.
+
+## Instructions for Specific Identity Providers
+
+Learn how to update your existing SAML configuration with the following IdPs:
+
+- [Okta](#okta)
+- [Azure AD](#azure-ad)
+- [Duo](#duo)
+- [Google](#google)
+- [OneLogin](#onelogin)
+
+### Okta
+
+<!--If you're using a preconfigured [Cobalt app](https://www.okta.com/integrations/cobalt/) in Okta, you only need to update the **Slug** value. To copy your organization's slug, in Cobalt, go to **Settings** > **General**.-->
+
+To update your existing SAML configuration with Okta:
+
+1. In Okta, go to your SAML application for Cobalt. On the **General** tab, under **SAML Settings**, select **Edit**, then select **Next**.
```

Comment:
they will only see SAML settings if they setup the orginal app manually. if they are using the Cobalt application from the catalog the steps are different. (they need to add the org slug variable and then fields will update.